### ISSUE TITLE
[AzureAD] Add all users to specified group on Azure AD 

### ISSUE THREAD

**Issue by PrzemyslawKlys** [EvotecIT/PSAutomator#1]
Something along the lines of:
```powershell
$Users = Get-AzureADUser -All $true
foreach ($User in $Users) {
    Add-AzureADGroupMember -ObjectId 'Self.pwd.reset' -RefObjectId $User.UserPrincipalName
}
```

**Comment by M1kep** [comment]
Out of curiosity, would this function be used for environments that don’t have AzureAD Sync?

**Comment by PrzemyslawKlys** [comment]
Yes. Not everyone has AD Sync and sometimes you may want to have synchronization of Office 365 group only with a group in your AD. There are multiple ideas that I have for this but my main problem is how to design the visual part (as in how to give user choice without complicating things. 

I have temporarily suspended work on PSAutomator because I hit a problem where I don't know the direction I want to take for the command names and their relations to each other. Feel free to make your suggestion :-)

**Comment by M1kep** [comment]
Fast response! That makes sense. I’ll take a look around and see if there’s anything I can help with. Lots of great modules in your blog post!
